app/extensions/brave/locales/en-US/menu.properties

```diff
@@ -43,10 +43,12 @@ home=Home
 back=Back
 forward=Forward
 reopenLastClosedWindow=Reopen Last Closed Window
-showAllHistory=Show All History
+showAllHistory=Show Full History
```

Comment:
For now just leave out the word `Full` since app state sites data is truncated to most recent 500.